### PR TITLE
Prog.ProgDVB version 7.41.4

### PR DESCRIPTION
**Pull request by chausner**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/23515)

### CODE REVIEW

File: manifests/p/Prog/ProgDVB/7.41.4/Prog.ProgDVB.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Prog.ProgDVB
+PackageVersion: 7.41.4
```
Comment (OfficialEsco) [inline manifests/p/Prog/ProgDVB/7.41.4/Prog.ProgDVB.installer.yaml:5]:
```suggestion
PackageVersion: 7.x
```

File: manifests/p/Prog/ProgDVB/7.41.4/Prog.ProgDVB.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Prog.ProgDVB
+PackageVersion: 7.41.4
```
Comment (OfficialEsco) [inline manifests/p/Prog/ProgDVB/7.41.4/Prog.ProgDVB.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 7.x
```

File: manifests/p/Prog/ProgDVB/7.41.4/Prog.ProgDVB.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Prog.ProgDVB
+PackageVersion: 7.41.4
```
Comment (OfficialEsco) [inline manifests/p/Prog/ProgDVB/7.41.4/Prog.ProgDVB.yaml:5]:
```suggestion
PackageVersion: 7.x
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
```
Name            Publisher Version ProductCode
----            --------- ------- -----------
ProgDVB x64 x64 Prog      7.x     ProgDVB
```
image

**Review by heroku-miraheze — CHANGES_REQUESTED** [review]
7.x??? Please fix and then we will approve it.

**Comment by OfficialEsco** [comment]
Well, are you the only user of this software?.. Contact the software developer about it.
This might get a workaround in manifest schema v1.1.0

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/23515#pullrequestreview-720959387

**Pull request closed without merging**